plugin-k8s/src/cgroup_v2.rs

```diff
@@ -40,27 +59,68 @@ pub fn is_accessible_dir(path: &Path) -> bool {
     path.is_dir()
 }
 
-
 /// Returns a Vector of CgroupV2MetricFile associated to pods availables under a given directory.
-fn list_metric_file_in_dir(root_directory_path: &Path) -> anyhow::Result<Vec<CgroupV2MetricFile>> {
+fn list_metric_file_in_dir(
```

Comment:
```suggestion
/// Returns a Vector of CgroupV2MetricFile associated to pods available under a given directory.
```

Reply:
I don't understand what you want me to change

Reply:
I don't either :thinking: